lib/ModelMixins/XmlRequestMixin.ts

```diff
@@ -0,0 +1,28 @@
+import Constructor from "../Core/Constructor";
+import isDefined from "../Core/isDefined";
+
+const loadXML = require("../Core/loadXML");
+const loadWithXhr = require("../Core/loadWithXhr");
+
+export default function XmlRequestMixin<T extends Constructor<any>>(Base: T) {
+  class XmlRequestMixin extends Base {
```

Comment:
I did this because I wanted to an easy way to stub network calls in specs. I had used it in a couple of places, so I thought of extracting to a mixin. But yeah, it doesn't feel right. Ideally we need a `Network` namespace which includes all the `loadX` functions, then we can stub them easily in the specs.

I'll just remove the `mixin` for now.

Reply:
Interesting. How does the mixin make it easier to stub network calls?

Reply:
That's simple, nothing complicated: https://github.com/TerriaJS/terriajs/blob/mobx-wps/test/Models/WebProcessingServiceCatalogFunctionSpec.ts#L116